loki/expression/symbols.py

```diff
@@ -1575,3 +1577,39 @@ def initial(self):
 
     mapper_method = intern('map_c_dereference')
     make_stringifier = loki_make_stringifier
+
+def resolve_type_bound_var(name):
```

Comment:
I have placed this here instead of `loki/tools/util.py` because that created a circular dependency.